README.md

```diff
@@ -52,31 +52,21 @@ can't be determined it will fail.
-| `url`                     | Connection string for the Postgres database.                                                                                                                                           | true     |               |
-| `table`                   | List of table names to read from, separated by comma. Example: `"employees,offices,payments"`                                                                                          | true     |               |
-| `key`                     | List of Key column names per table, separated by comma. Example:`"table1:key1,table2:key2"`, if not supplied, the table primary key will be used as the `'Key'` field for the records. | false    |               |
-| `snapshotMode`            | Whether or not the plugin will take a snapshot of the entire table before starting cdc mode (allowed values: `initial` or `never`).                                                    | false    | `initial`     |
-| `cdcMode`                 | Determines the CDC mode (allowed values: `auto`, `logrepl` or `long_polling`).                                                                                                         | false    | `auto`        |
-| `logrepl.publicationName` | Name of the publication to listen for WAL events.                                                                                                                                      | false    | `conduitpub`  |
-| `logrepl.slotName`        | Name of the slot opened for replication events.                                                                                                                                        | false    | `conduitslot` |
-
+| name                      | description                                                                                                                                                                                | required | default       |
+|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| `url`                     | Connection string for the Postgres database.                                                                                                                                               | true     |               |
+| `tables`                  | List of table names to read from, separated by comma. Example: `"employees,offices,payments"`. Using `*` will read from all public tables.                                                 | true     |               |
+| `key`                     | List of Key column names per table, separated by comma. Example:`"table1:key1,table2:key2"`, if not supplied, the table(s) primary keys will be used as the `'Key'` field for the records. | false    |               |
```

Comment:
i.e. this assumes one key per table?

Reply:
Will address on https://github.com/ConduitIO/conduit-connector-postgres/pull/135. However, as discussed today at stand up, we're removing `key` altogether since we're going to rely on primary keys automatically.